be/src/util/hash_util.hpp

```diff
@@ -400,4 +401,15 @@ struct hash<std::pair<starrocks::TUniqueId, int64_t>> {
     }
 };
 
+template <typename T>
+struct hash<std::vector<T>> {
+    size_t operator()(const std::vector<T>& ts) const noexcept {
+        std::size_t hash_value = 0; // seed
+        for (auto& t : ts) {
+            boost::hash_combine<T>(hash_value, t);
```

Comment:
if two vector<t> have different order but same element, the hash value will be same.

Reply:
I just tested it is ok. https://godbolt.org/z/Mr8TfsYsh